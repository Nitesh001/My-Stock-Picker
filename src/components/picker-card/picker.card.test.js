import React from 'react';
import renderer from 'react-test-renderer';
import PickerCard from './picker.card';

test('Card shows title and value', () => {
  const component = renderer.create(
    <PickerCard title="title" value="value" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});