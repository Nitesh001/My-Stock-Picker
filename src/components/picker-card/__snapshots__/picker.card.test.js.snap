// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card shows title and value 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      title
    </div>
    <p
      className="card-text"
    >
      value
    </p>
  </div>
</div>
`;
